csuntechlab/iSTART
# STR-65 -- FE: Create loading feedback for login

# Description
adds visual loading feedback when a user logs in
  
# Testing
1. login using user of choice
2. make sure loading icon comes up and you cant use the inputs or press the button 
3. enjoy your monday

# installation
yarn install if you havent installed progress bars yet
